### PR TITLE
feat(tx-bundling): update button content

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Update button content based on discussion captured on [this figma](https://www.figma.com/file/pYXF21s8v1S1LfxpWvudbM/2023---TX-Bundling-v1?type=design&node-id=0-1&t=AgeIUSNIEmrgiaSv-0)

# To Test

1. On Safe app Swap, pick a token that needs approval as sell
* Text should be 
image

2. Click on the Approve & Swap button
* The confirm button text should be 
image

3. Turn on expert mode
* The confirm button text should be 
image

4. Repeat steps for limit orders
* Step `1` text should be
image

* Step `2` (confirmation) text should be
image

* Step `3` (expert) text should be
image

Regular (non bundle) operations should remain with the current button text labels

### CODE REVIEW

File: src/components/swap/ConfirmSwapModal/ConfirmSwapModalMod.tsx
Comment (alfetopito) [inline src/components/swap/ConfirmSwapModal/ConfirmSwapModalMod.tsx:1]: Removed mod file

File: src/components/swap/SwapModalFooter/SwapModalFooterMod.tsx
Comment (alfetopito) [inline src/components/swap/SwapModalFooter/SwapModalFooterMod.tsx:1]: Removed mod file

File: src/components/swap/ConfirmSwapModal/index.tsx [hunk 1]
```diff
@@ -1,10 +1,126 @@
...
+  const isSafeApprovalBundle = useIsSafeApprovalBundle(trade?.inputAmount)
+  const buttonText = useMemo(
+    () =>
+      isSafeApprovalBundle ? (
+        <>
+          Confirm (Approve&nbsp;{<TokenSymbol token={trade?.inputAmount?.currency.wrapped} length={6} />}&nbsp;and Swap)
+        </>
+      ) : undefined,
+    [isSafeApprovalBundle, trade?.inputAmount?.currency.wrapped]
+  )
```
Comment (alfetopito) [inline src/components/swap/ConfirmSwapModal/index.tsx:83]: The actual change, plus line `91` where `buttonText` is added to component props

File: src/components/swap/SwapModalFooter/index.tsx [hunk 2]
```diff
@@ -9,10 +14,31 @@ const Wrapper = styled.div`
...
+          style={{ margin: '10px 0 0 0' }}
+          id="confirm-swap-or-send"
+          buttonSize={ButtonSize.BIG}
+        >
+          <Text fontSize={20} fontWeight={500}>
+            <Trans>{buttonText || 'Confirm Swap'}</Trans>
+          </Text>
+        </ButtonError>
+
+        {swapErrorMessage ? <SwapCallbackError error={swapErrorMessage} /> : null}
```
Comment (alfetopito) [inline src/components/swap/SwapModalFooter/index.tsx:40]: The actual change to this component

**Review by elena-zh — APPROVED** [review]

**Review by anxolin — APPROVED** [review]

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**